app/assets/stylesheets/views/_landing_page/card.scss

```diff
@@ -0,0 +1,44 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.app-b-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background-color: govuk-colour("dark-blue");
+  color: govuk-colour("white");
+}
+
+.app-b-card__textbox {
+  padding: govuk-spacing(6) govuk-spacing(7);;
+  background-color: govuk-colour("dark-blue");
+  color: govuk-colour("white");
+
+  .gem-c-govspeak h2 {
+    @include govuk-font($size: 27, $weight: bold, $line-height: 34px );
+  }
+
+}
+
+.app-b-card__figure {
+  border: 1px solid govuk-colour("mid-grey");
```

Comment:
Non-blocking: this mimics the design exactly but I wonder if the top border is needed here?

Reply:
Funnily enough I did originally implement that as I thought it looked cleaner, then removed it to match the design. I've removed it again now.